source/crud/tests/v2/bulkWrite-arrayFilters-clientError.yml

```diff
@@ -0,0 +1,35 @@
+runOn:
+    -
```

Comment:
I'm not sure if it's significant for YAML, but in `tests:` later on you're using a two-space indent, while `runOn:` and `data:` uses four. I think we can use two throughout the file.

I realize this might have been copied from [`bulkWrite-arrayFilters.yml`](https://github.com/mongodb/specifications/blob/master/source/crud/tests/v2/bulkWrite-arrayFilters.yml), which has the same inconsistency. No objections if you want to fix that in a separate commit within this PR.

Reply:
You are correct that I copy/pasted that.  I'm fixing the spacing in both files.